app/views/course_participants/list.html.erb

```diff
@@ -21,7 +21,6 @@
                           :action=>'start',
                           :model => 'Participant',
                           :title => @title+' Participants',
```

Comment:
Why is this changed?

Reply:
The required_import_fields is now handled by the model itself and those fields are pulled into the import file controller's view page.